pkg/helm/chart_info.go

```diff
@@ -268,9 +270,10 @@ func parseChartValuesFileComments(chartDirectory string, values *yaml.Node, lint
 		// If we've already found a values comment, on the next line try and parse a custom default value. If we find one
```

Comment:
Update this comment to reflect that we can also continue and parse a section comment now. Add a note about the continuation regex match as well, that should have been done before.